synapse/federation/sender/per_destination_queue.py

```diff
@@ -248,6 +238,12 @@ def _transaction_transmission_loop(self):
                         )
                     )
 
+                pending_edus.extend(device_message_edus)
+                pending_edus.extend(self._pop_pending_edus(100 - len(pending_edus)))
```

Comment:
this `100` number is starting to appear in a lot of places. Could I suggest creating a constant `MAX_EDUS_PER_TRANSACTION` in this file?